products/cloudiot/api.yaml

```diff
@@ -0,0 +1,194 @@
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: |
+          The resource path name. For example,
+          `projects/example-proj/locations/us-central1/registries/my-registry`.
+      - !ruby/object:Api::Type::Array
+        name: 'eventNotificationConfigs'
+        description: |
+          List of configurations for event notifications, such as PubSub topics
+          to publish device events to.
+        max_size: 10
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: 'subfolderMatches'
+              description: |
+                If the subfolder name matches this string exactly, this
+                configuration will be used. The string must not include the
+                leading '/' character. If empty, all strings are matched. Empty
+                value can only be used for the last `event_notification_configs`
+                item.
+            - !ruby/object:Api::Type::String
+              name: 'pubsubTopicName'
+              required: true
+              description: |
+                PubSub topic name to publish device events.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'stateNotificationConfig'
+        description: |
+          A PubSub topic to publish device state updates.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'pubsubTopicName'
+            required: true
+            description: |
+              PubSub topic name to publish device state updates.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'mqttConfig'
+        description: |
+          Activate or deactivate MQTT.
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: 'mqttEnabledState'
+            description: |
+              The field allows `MQTT_ENABLED` or `MQTT_DISABLED`
+            required: true
+            values:
+              - :MQTT_ENABLED
+              - :MQTT_DISABLED
+      - !ruby/object:Api::Type::NestedObject
+        name: 'httpConfig'
+        description: |
+          Activate or deactivate HTTP.
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: 'httpEnabledState'
+            required: true
+            description: |
+              The field allows `HTTP_ENABLED` or `HTTP_DISABLED`.
+            values:
+              - :HTTP_ENABLED
+              - :HTTP_DISABLED
+      - !ruby/object:Api::Type::Enum
+        name: 'logLevel'
+        description: |
+          The default logging verbosity for activity from devices in this
+          registry. Specifies which events should be written to logs. For
+          example, if the LogLevel is ERROR, only events that terminate in
+          errors will be logged. LogLevel is inclusive; enabling INFO logging
+          will also enable ERROR logging.
+
+          The field allows `NONE`, `ERROR`, `INFO`, `DEBUG`.
```

Comment:
We recently started documenting enum values automatically, so this can be removed (here and elsewhere)
You can see what it looks like now at https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-3357-old..auto-pr-3357#diff-f093bf8c1e1d785bca277d59b605db0bR155-R168

Reply:
Oh nice!

Fixed!.